components/style/properties/properties.mako.rs

```diff
@@ -1181,13 +1181,21 @@ impl ComputedValues {
     #[inline]
     pub fn min_inline_size(&self) -> computed::LengthOrPercentage {
         let position_style = self.get_position();
-        if self.writing_mode.is_vertical() { position_style.min_height } else { position_style.min_width }
+        let size = if self.writing_mode.is_vertical() { position_style.min_height } else { position_style.min_width };
+        match size {
+            computed::MinLength::LengthOrPercentage(lop) => lop,
+            _ => panic!("layout doesn't work!")
```

Comment:
Of course, this totally broke because `position_style.min_height` and `position_style.min_width` are both `MinLength` now, instead of `LengthOrPercentage`. I believe that @Manishearth told me in #13821 that I will need to do something to make sure that Servo doesn't parse the keywords. Is that how I'm supposed to handle this case? I'm sure that what I have is completely wrong...